fairlearn/postprocessing/_thresholder.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.

```

Comment:
In new files we can remove the Microsoft part and just write Fairlearn contributors. 🙂 I'm sure that's a bit confusing, sorry for that!